cwltool/schemas/v1.1/conformance_tests.yaml

```diff
@@ -2598,6 +2598,17 @@
   doc: Test an anonymous enum inside an array inside a record, SchemaDefRequirement
   tags: [command_line_tool, schema_def]
 
+
+- job: tests/empty.json
+  tool: tests/fail-unconnected-expressiontool.cwl
+  label: wf_step_access_undeclared_param_expressiontool
+  id: 198
+  doc: >-
+    Test that parameters that don't appear in the `run` process
+    inputs are not present in the input object used to run the expressiontool.
+  should_fail: true
+  tags: [ required, workflow ]
+
```

Comment:
The CI doesn't run this copy of the tests, it makes a new checkout from GitHub https://github.com/common-workflow-language/cwltool/blob/7a058fed2987bc0046ffbc9a2b8a39b5b4fec6e1/conformance-test.sh#L41

Reply:
@mr-c Thanks

I will create a PullRequest about this test case, 

Repo:
https://github.com/common-workflow-language/cwl-v1.2
Branch:
`1.2.1_proposed`

Reply:
Thanks, you can also keep it in this PR, just move the test scripts to the the test directory and add an entry to test_examples.py

Reply:
Can you move this file to the `tests` directory, or a sub-directory?